playground/android/app/build.gradle

```diff
@@ -57,20 +50,20 @@ dependencies {
     implementation 'com.google.android.material:material:1.4.0'
```

Comment:
I think we can safely delete the line above the one I highlighted:

`implementation fileTree(dir: 'libs', include: ['*.jar'])`